en/getting_started/README.md

```diff
@@ -25,22 +25,35 @@ To get the source files:
 
 ### Native Builds
 
-*QGroundControl* builds are supported for OSX, Linux, Windows, iOS and Android. *QGroundControl* uses [Qt](http://www.qt.io) as its cross-platform support library and uses [QtCreator](http://doc.qt.io/qtcreator/index.html) as its default build environment.
+*QGroundControl* builds are supported for macOS, Linux, Windows, iOS and Android. *QGroundControl* uses [Qt](http://www.qt.io) as its cross-platform support library and uses [QtCreator](http://doc.qt.io/qtcreator/index.html) as its default build environment.
 
-- **OSX:** OSX 10.7 or higher, 64 bit, clang compiler
+- **macOS:** v10.11 or higher
 - **Ubuntu:** 64 bit, gcc compiler
 - **Windows:** Vista or higher, 32 bit, [Visual Studio 2015 compiler](http://www.visualstudio.com/downloads/download-visual-studio-vs#d-express-windows-desktop)
```

Comment:
@DonLakeFlyer Where it says ", 32 bit, [Visual Studio 2015 compiler]", does it mean ", [Visual Studio 2015 compiler] (32 bit)" - ie the common after 32 bit is confusing